packages/@ember/-internals/glimmer/lib/utils/bindings.ts

```diff
@@ -118,6 +118,14 @@ export const AttributeBinding = {
 const DISPLAY_NONE = 'display: none;';
 const SAFE_DISPLAY_NONE = htmlSafe(DISPLAY_NONE);
 
+function emitIsVisibleDeprecation() {
+  deprecate('`isVisible` is deprecated', false, {
```

Comment:
I'd like to include `${this}` (the component instances toString) to give the person a fighting chance to find where to look for the offending component...